packages/acter_notifify/lib/model/push_styles.dart

```diff
@@ -33,3 +33,17 @@ enum PushStyles {
 
   final String emoji;
 }
+
+//LIST OF PUSH STYLE EMOJIS
+enum SpaceObjectTypes {
```

Comment:
as said before. I don't think we should use emojis (as also screen readers read them out) wherever we can have actual icons.